lib/datatype/indexer/index.js

```diff
@@ -82,6 +76,7 @@ export class Indexer {
           typeof value === 'object' &&
           ['links', 'forks'].includes(key) === false
         ) {
+          /* @ts-ignore */
```

Comment:
I realize this change is not in this PR, but just spotting this now and wondering how we know to parse these values on read?

Reply:
Yeah that's currently in the sqlite wrapper: https://github.com/digidem/mapeo-core-next/pull/38/files/82fb39f5097565d9e9c8238c0f7d7be01b2b15f2#diff-f391aa92bfbc3da0b7157d69e715a17508f09dc1fbd520a49befb3b446ede72eR32

Ideally the mapeo-schema encode function would handle this part. But for parsing json from sqlite maybe we need to read the json schema and parse values based on that.